patches/544307D1 - DEAD OR ALIVE 4 (PROTYTPE).patch.toml

```diff
@@ -0,0 +1,13 @@
+title_name = "DEAD OR ALIVE 4"
+title_id = "544307D1" # TC-2001
+hash = "ED5C2458F214AB36" # default.xex
+#media_id = "00000000"
```

Comment:
Is this the real media ID?

Reply:
Fixed the filename typo. The Media ID is correct; it hadn't been assigned yet. Not sure if it's worth including.  

`i> 00004ECC Module \Device\Harddisk0\Partition1\doa4.exe:
Module Hash: ED5C2458F214AB36
    Module Flags: 00000001
Security Header:
     Image Flags: 00000000
    Load Address: 82000000
      Image Size: 01190000
    Export Table: 00000000
Optional Header Count: 13
  XEX_HEADER_RESOURCE_INFO:
    544307D1 83170000-8318857F, 99711b
  XEX_HEADER_FILE_FORMAT_INFO (TODO):
  XEX_HEADER_ORIGINAL_BASE_ADDRESS: 00400000
  XEX_HEADER_ENTRY_POINT: 825708D8
  XEX_HEADER_IMAGE_BASE_ADDRESS: 82000000
  XEX_HEADER_IMPORT_LIBRARIES:
    xam.xex - 102 imports
      Version: 0.0.2099.32
      Min Version: 0.0.1861.32
    xboxkrnl.exe - 300 imports
      Version: 0.0.2099.32
      Min Version: 0.0.1861.32
  XEX_HEADER_CHECKSUM_TIMESTAMP:
    Checksum : 00C29F3F
    Time Stamp: 433D906B - Fri Sep 30 13:22:19 2005
  XEX_HEADER_STATIC_LIBRARIES:
    XAPILIB  : 2.0.2099.3
    XRTLLIB  : 2.0.2099.3
    D3D9     : 2.0.2099.3
    D3DX9    : 2.0.2099.3
    XGRAPHC  : 2.0.2099.3
    XBOXKRNL : 2.0.2099.3
    XNET     : 2.0.2099.3
    XONLINE  : 2.0.2099.3
    XAUD     : 2.0.2099.3
    XACT     : 2.0.2099.3
    LIBCMT   : 2.0.2099.3
    XHV      : 2.0.2099.3
    X3DAUD   : 2.0.2099.3
  XEX_HEADER_TLS_INFO:
          Slot Count: 64
    Raw Data Address: 00000000
           Data Size: 0
       Raw Data Size: 0
  XEX_HEADER_DEFAULT_STACK_SIZE: 262144
  XEX_HEADER_SYSTEM_FLAGS: 00000200
  XEX_HEADER_EXECUTION_INFO:
       Media ID: 00000000
       Title ID: 544307D1
    Savegame ID: 544307D1
    Disc Number / Total: 0 / 0
  XEX_HEADER_LAN_KEY: 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00`